releng/gen.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/redhat-developer/tekton-jsongenerator/jsonschema"
+	v1alpha1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	resource "github.com/tektoncd/pipeline/pkg/apis/resource/v1alpha1"
+	triggers "github.com/tektoncd/triggers/pkg/apis/triggers/v1alpha1"
+	"os"
+	"encoding/json"
+)
+
+
+func dump(v interface{}, apiVersion string, kind string, list bool) {
```

Comment:
It looks to me that you use the bool `list` only to create the correct `realKind` (so if you pass `PipelineRun` as kind and list is true the `realKind` will be `PipelineRunList`), so why don't you directly pass the list kind, such as `PipelineRunList`?